### PR TITLE
Change DataServlet class to return JSON of shows.  Then display them in html.

### PR DESCRIPTION
**Pull request by josefj1519**
Assignee: LiRq95

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -13,20 +13,31 @@
...
+/* Servlet that returns some example content.*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Josef!");
+    ArrayList<String> shows = new ArrayList<String>();
```
Comment (LiRq95) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: nit: Generally a good idea to use the interface as the type (List) as common utility methods with Collections don't return Arraylist. Also, the second 'String' doesn't need to be added as it's inferred, so could just be '= new ArrayList<>();'.
You could also probably also instantiate all of its values in 1 line without losing clarity with:
`List<String> list = Arrays.asList("a", "b", "c");`

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -13,20 +13,31 @@
...
+    shows.add("Avatar: The Last Airbender");
+    shows.add("Arrested Development");
+    shows.add("Breaking Bad");
+    response.setContentType("application/json;");
+    response.getWriter().println(convertToJsonUsingGson(shows));
+  }
+  
+  private String convertToJsonUsingGson(ArrayList<String> shows) {
```
Comment (LiRq95) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:38]: Likewise as above, good idea to use List

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -13,20 +13,31 @@
...
+    Gson gson = new Gson();
+    String json = gson.toJson(shows);
+    return json;
```
Comment (LiRq95) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:41]: nit: This and the line above can be 1 line without sacrificing too much clarity.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (LiRq95) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Another note, if this list will never change, it's an even better idea to use an ImmutableList to make sure of that.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 3 above)
Comment (josefj1519) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Done.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (josefj1519) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Done.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (josefj1519) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Fixed and noted.  Later on the list will change.

**Pull request merged**